src/App.test.js

```diff
@@ -0,0 +1,8 @@
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
+
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
```

Comment:
- [ ] Please consider removing commented codes as it will make your codebase messy. Remember, if you ever want to see the code you have written once you can use your commit history.